content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -76,6 +76,27 @@ To account for modern web applications, loading time watches for network request
 
 The RUM SDK automatically monitors frameworks that rely on hash (`#`) navigation. The SDK watches for `HashChangeEvent` and issues a new view. Events coming from an HTML anchor tag which do not affect the current view context are ignored.
 
+## Add your own performance timing
+On top of RUM's default performance timing, you may measure where your application is spending its time with greater flexibility. The `addTiming` API provides you with a simple way to add extra performance timing. For example, you can add a timing when your hero image has appeared:
+
+```html
+<html>
+  <body>
+    <img onload="DD_RUM.addTiming('hero_image')" src="/path/to/img.png" />
+  </body>
+</html>
+```
+
+Or when users first scroll:
+
+```javascript
+document.addEventListener("scroll", function handler(e) {
+    e.currentTarget.removeEventListener(e.type, handler); //Remove the event listener so that it only triggers once
+    DD_RUM.addTiming('first_scroll');
+});
+```
+
```

Comment:
Should we add something about facet creation?
like RUM Explorer > Add a facet > @view.custom_timings.first_scroll